internal/version/version.go

```diff
@@ -44,11 +48,8 @@ var (
 // the official release process (as opposed to a user just running "go build",
 // or running in a CI environment, or something else).
 func IsReleaseBuild() bool {
-	// "source" is a sentinel value returned from buildinfo.Version() when the
-	// build metadata is not present. The build metadata is filled in with the
-	// "-X" flag during release builds, so if it's not present, this binary is
-	// not a release build.
-	return Version != "source"
+	// Binary from release build should follow semver format.
+	return semver.IsValid(Version)
```

Comment:
I checked the code, the Version here should starts with "v". When we released v0.0.6-rc, we'd better double check it.